typings/index.flow.js

```diff
@@ -202,6 +202,11 @@ interface UnsafeByPropsQueries {
     | Array<ReactTestInstance>
     | [];
 }
+
+interface ByRoleOption {
+  name: string;
```

Comment:
The option should be optional

```suggestion
  name?: string;
```